posts/2024-01-31-Performance-Doris_Amoakohene/index.qmd

```diff
@@ -0,0 +1,248 @@
+---
+title: "Visualizing Performance Regression of data.table with Atime"
+author: "Doris Afriyie Amoakohene"
+date: "2024-02-04"
+output: html_document
+---
+
+
+Since August 2023, I have been working on performance testing, which could be useful for expanding the open-source ecosystem around data.table package in R.
+This could increase confidence in code contributions by ensuring the sustained efficiency of the data.table package. Consequently, it becomes essential to enhance the package's performance and deliver reliable results to its users.
```

Comment:
can you please clarify "Consequently, it becomes essential to enhance the package's performance and deliver reliable results to its users."

Reply:
I want to say that its necessary to improve or enhance the performance of the package so that it can consistently provide reliable and satisfactory results to its users, but I have already spoke about it, so i will leave that out. @tdhock